### ISSUE TITLE
azure-functions-core-tools@3.0.3528: hash check failed

### ISSUE THREAD

**Issue by dam5s** [ScoopInstaller/Main#2177]
Output from my installation:

```
 ✗ scoop install azure-functions-core-tools
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'azure-functions-core-tools' (3.0.3528) [64bit]
Starting download with aria2 ...
Invoke-Expression: C:\Users\Damo\scoop\apps\scoop\current\lib\install.ps1:287
Line |
 287 |          Invoke-Expression $aria2 | ForEach-Object {
     |          ~~~~~~~~~~~~~~~~~~~~~~~~
     | This script contains malicious content and has been blocked by your antivirus software.


Checking hash of Azure.Functions.Cli.win-x64.3.0.3528.zip ... Get-FileHash: C:\Users\Damo\scoop\apps\scoop\current\lib\install.ps1:691
Line |
 691 |  …          return (Get-FileHash -Path $file -Algorithm $algname).Hash.T …
     |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\Users\Damo\scoop\cache\azure-functions-core-tools#3.0.3528#https_github.com_Azure_azure-functions-core-tools_releases_download_3.0.3528_Azure.Functions.Cli.win-x64.3.0.3528.zip_dl.7z' because it does not exist.

ERROR You cannot call a method on a null-valued expression.
ERROR Hash check failed!
App:         main/azure-functions-core-tools
URL:         https://github.com/Azure/azure-functions-core-tools/releases/download/3.0.3528/Azure.Functions.Cli.win-x64.3.0.3528.zip#/dl.7z
Expected:    bfd0359b912c498f70d34620bfa2712735fa1482c9d453edcbd7c65f98ebe534
Actual:

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=azure-functions-core-tools%403.0.3528%3a+hash+check+failed
```

Scoop version output
```
Current Scoop version:
3d67b7d3 (HEAD -> master, origin/master, origin/HEAD) Fixed trailing whitespace issue (#4307)

'extras' bucket:
98006c135 (HEAD -> master, origin/master, origin/HEAD) tumblthree: Update to version 1.5.0

'java' bucket:
45df534f (HEAD -> master, origin/master, origin/HEAD) graalvm-nightly: Update to version 21.2.0-dev-20210513_1950

'main' bucket:
e321b6d0a (HEAD -> master, origin/master, origin/HEAD) okteto: Update to version 1.12.12
```

**Comment by dam5s** [comment]
It looks like aria2 was the issue. Disabled it. Good bot, thanks.
